<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clipScene</class>
 <widget class="QDialog" name="clipScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Set Records ID</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="3" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CROPPED IMAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Image Path</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGraphicsView" name="graphicsViewClipped"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ORIGINAL IMAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Date &amp; Time:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Image Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Latitude:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Longitude:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Depth:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string> Length [mm]:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Set Primary Substrate:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Set Secondary Substrate:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Species:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Additional Species:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Save Cropped Image:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QDateTimeEdit" name="dateTimeEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="imageNameLineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="latitudeLineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="longitudeLineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="depthLineEdit"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lengthLineEdit"/>
           </item>
           <item>
            <widget class="QComboBox" name="primarySubstrateComboBox">
             <item>
              <property name="text">
               <string>--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rock</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Boulder</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cobble</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pebble</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sand</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mud</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="secondarySubstrateComboBox">
             <item>
              <property name="text">
               <string>--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rock</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Boulder</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cobble</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pebble</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sand</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mud</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="speciesComboBox">
             <item>
              <property name="text">
               <string>--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Red Cell</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>White Cell</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Plaster</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="addParamComboBox">
             <item>
              <property name="text">
               <string>--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Red Cell First Type</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>White Cell First Type</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Red Cell Second Type</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Whte Cell Second Type</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="noCBox">
             <property name="text">
              <string>Save</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>First Point XYZ</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>X-Coord</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>Y-Coord</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLineEdit" name="firstXCoord"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="firstYCoord"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Second Point XYZ </string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>X-Coord</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Y-Coord</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLineEdit" name="secondXCoord"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="secondYCoord"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="pathLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonPath">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="res/res.qrc">
             <normaloff>:/res/images.png</normaloff>:/res/images.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="4">
        <widget class="QLabel" name="pathImage">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>96</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CLICKED POINT(S)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Coordinate(s)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CLASSIFIED IMAGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGraphicsView" name="graphicsViewBBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="loadBtnBBox">
             <property name="text">
              <string>Load Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;COORDINATE(S) OF BOUNDING BOX&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>A (x,y):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>B (x,y):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>C (x,y):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>D (x,y):</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="ACoord">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="BCoord">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="CCoord">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DCoord">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>RGB</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Blue Intensity (B)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Mean</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="blueLineIntensityEdit"/>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="redImg">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Green Intensity (G)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Mean</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="greenImg">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="greenLineIntensityEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Red Intensity (R)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Mean</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="redLineIntensityEdit"/>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="blueImg">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptBtn">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
